static-site/src/components/ListResources/Showcase.tsx

```diff
@@ -115,19 +126,19 @@ const Byline = styled.div`
  * which the filters are valid given the resources known to the resource listing
  * UI
  */
-export const useShowcaseCards = (cards, resources) => {
-  const [restrictedCards, setRestrictedCards] = useState([]);
+export const useShowcaseCards = (cards?: Card[], groups?: Group[]) => {
```

Comment:
> Functions don't have to be anonymous closures assigned to named variables:

In general I'm +1 for using named functions instead of anonymous funcs assigned to named variables. (All functions are closures in JS, right? Not just the arrow funcs.)

But it's funny, because the arrow func syntax reminds me of Haskell and in the context of [doing functional composition in JS](https://github.com/nextstrain/nextstrain.org/blob/08db0bd73841c8e0ef798e679167fb11ae014664/src/endpoints/sources.js#L83-L120), the arrow syntax feels way more natural to me. So "use named functions" isn't an absolute thing for me.